easybuild/easyconfigs/o/OpenMPI/OpenMPI-5.0.3-GCC-13.3.0.eb

```diff
@@ -33,6 +33,9 @@ dependencies = [
 
 # CUDA related patches and custom configure option can be removed if CUDA support isn't wanted.
 preconfigopts = 'gcc -Iopal/mca/cuda/include -shared opal/mca/cuda/lib/cuda.c -o opal/mca/cuda/lib/libcuda.so && '
-configopts = '--with-cuda=%(start_dir)s/opal/mca/cuda --with-show-load-errors=no '
+configopts = '--with-cuda=%(start_dir)s/opal/mca/cuda --with-show-load-errors=no'
+# Do not pick up the system library automatically
+configopts += ' --without-xpmem'
+buildopts = 'V=1'
```

Comment:
If we want verbose build output, that should go into the custom easyblocks we have for OpenMPI?

Reply:
Might be better, yes. I copied that from other versions as it made sense for this issue to verify the commands.

Reply:
I opened https://github.com/easybuilders/easybuild-easyblocks/pull/3724 for this and will update this PR to remove `V=1` from all easyconfigs for consistency